src/h5web/vis-packs/core/utils.ts

```diff
@@ -387,3 +389,15 @@ export function createArrayFromView<T>(view: NdArray<T>): NdArray<T> {
 
   return array;
 }
+
+export function hasImageAttributes(dataset: Dataset): boolean {
+  const classAttr = getAttributeValue(dataset, 'CLASS');
+  const versionAttr = getAttributeValue(dataset, 'IMAGE_VERSION');
+
+  return (
+    typeof classAttr === 'string' &&
+    classAttr === 'IMAGE' &&
+    typeof versionAttr === 'string' &&
+    versionAttr === '1.2'
```

Comment:
I'm not convinced the version check is required. This seems like the kind of attribute people might forget. I'd be more for following the same guideline as we do for NeXus of checking the least amount of stuff possible (while still failing on critical syntax errors or unsupported features like `IMAGE_SUBCLASS=IMAGE_BITMAP`)